.github/workflows/main.yml

```
@@ -52,7 +52,7 @@ jobs:
         run: |
           conda create --yes -n tester python=${{ matrix.python-version }} pip numpy 'scipy>=0.17.0' matplotlib pandas flake8 pep8 jupyter coverage cython scikit-learn requests
           conda install -n tester -c conda-forge phantomjs --yes
-          conda run -n tester pip install sphinx numpydoc sphinx-bootstrap-theme
+          conda run -n tester pip install sphinx<4 numpydoc sphinx-bootstrap-theme
```

Thanks for catching, I think this is just missing quotes. And then the build should work 🤞 

```suggestion
          conda run -n tester pip install "sphinx<4" numpydoc sphinx-bootstrap-theme
```